l10n/en/products/relay/matrix.ftl

```diff
@@ -21,7 +21,7 @@ plan-matrix-heading-features = Features
 plan-matrix-heading-plan-free = Limited email protection
 plan-matrix-heading-plan-premium = Email protection
 plan-matrix-heading-plan-phones = Email & phone protection
-plan-matrix-heading-plan-bundle-2 = Add { -brand-name-vpn } protection
+plan-matrix-heading-plan-bundle-v2 = Add VPN protection
```

Comment:
**Suggestion** You should update the `matrix.html` file to include this new version of the string. This is what it currently shows:
![Screenshot 2023-07-28 at 3 52 11 PM](https://github.com/mozilla/bedrock/assets/42309026/4b56c21c-0f79-4faa-b210-5485bd0e0534)